### PR TITLE
Add requirements for Windows, Docker Desktop for Linux released

### LINKED ISSUE #1276: Install containers is missing system requirements
https://6.docs.plone.org/install/containers/index.html#system-requirements

### LINKED ISSUE #1468: Specify supported and unsupported OSes and alternatives for Windows users
### Plone Documentation versions affected

- [ ] 5.x
- [X] 6.x

### Description

Now that we use `make` everywhere, effectively Windows is no longer supported for Plone 6. We should specify that in system requirements under Install:

- https://6.docs.plone.org/install/containers/index.html#install-containers-index-system-requirements-label
- https://6.docs.plone.org/install/install-from-packages.html#install-packages-system-requirements-label

This sentence is a lie:

> Almost any modern operating system, including Linux, macOS, and **Windows**, but a UNIX-based operating system is recommended.

@tiberiuichim @sneridagh @jensens @mauritsvanrees @davisagli I could use your feedback and historical knowledge before I start making any changes.

Refs:
- https://github.com/plone/volto/issues/4564
- https://github.com/plone/documentation/issues/1276

### LINKED ISSUE #4564: Volto is unable to run on Windows
It seems that there's a lot of people that do development on Windows. 

The biggest problem, right now, is that it fails to run `make`.

### PR DESCRIPTION
**Pull request by stevepiercy**
Closes #1468 

Closes #1276

Closes https://github.com/plone/volto/issues/4564

If I missed anyone who should review these requirements, please let me know.

### CODE REVIEW

File: docs/install/containers/index.md [hunk 1]
```diff
@@ -36,21 +36,21 @@ Although there are many container engine tools for developing, managing, and run
...
-Install [Docker](https://docs.docker.com/get-docker/) for your operating system.
+Install [Docker Desktop](https://docs.docker.com/get-docker/) for your operating system.
 
 Docker Desktop includes all Docker tools.
-Both macOS and Windows use Docker Desktop.
-On some Linux distributions a beta version of Docker Desktop is available.
-If Docker Desktop is not available for your Linux distribution, you may still install Docker Engine and all of its tools.
-Consult [Get Docker](https://docs.docker.com/get-docker/) for details.
-
 {term}`Docker Compose` is one of the Docker tools that will be used throughout this documentation.
```
Comment (davisagli) [inline docs/install/containers/index.md]: I don't see any use of Docker Compose in the commands below.

File: docs/install/install-from-packages.md [hunk 2]
```diff
@@ -37,6 +43,15 @@ You may host multiple Plone sites on the same server.
 
 ### Pre-requisites for installation
 
+-   An operating system that runs all the pre-requisites.
+    Most UNIX-based operating systems are supported, including many Linux distributions, macOS, or WSL on Windows.
```
Comment (davisagli) [inline docs/install/install-from-packages.md]: WSL should probably be a link or glossary term, no? Imagine I'm a Windows user who does not know what WSL is; where do I go next?

Comment (stevepiercy) [reply docs/install/install-from-packages.md]: Fixed in 3bcdc9665ca2316216807fc3fb18dee00bb1b8d6. Thank you!

**Review by davisagli — DISMISSED** [review]

File: docs/install/containers/index.md (hunk 1 above)
Comment (stevepiercy) [reply docs/install/containers/index.md]: True, but there are other places in the containers install and Volto documentation that do. I could list all the docker tools and commands, but I think that would be better left to Docker to document. This should be more accurate:

s/throughout/in much of

Comment (stevepiercy) [reply docs/install/containers/index.md]: Fixed in 4125d2150da8e77a875318ea4c465798168a79d7

**Comment by stevepiercy** [comment]
I'm going to let this PR sit for a few days and marinate. This is a fairly big change.

File: docs/install/containers/index.md (hunk 1 above)
Comment (davisagli) [reply docs/install/containers/index.md]: @stevepiercy Got it, it made more sense once I read it again and realized it wasn't talking only about this page.

**Review by davisagli — APPROVED** [review]
I think this looks okay, but makes sense if you want to wait and get more opinions.

**Review by jensens — APPROVED** [review]

**Review by ksuess — APPROVED** [review]
LGTM

**Comment by stevepiercy** [comment]
With the overwhelming flood of GSoC aspirants, I'm going ahead and merging this PR.

Please feel free to add your comment, even though this is merged.

**Pull request merged**